core/modules/simpletest/tests/image.test

```diff
@@ -323,14 +322,14 @@ class ImageToolkitGdTestCase extends BackdropWebTestCase {
       $operations += array(
         'rotate_5' => array(
           'function' => 'rotate',
-          'arguments' => array(5, 0xFF00FF), // Fuchsia background.
+          'arguments' => array(5, '0xFF00FF'), // Fuchsia background.
```

Comment:
For some reason PHP is converting `0xFF00FF` into the integer `16711935` and that's what's throwing off the tests. All the functions down the line need to receive a value in the form `#FF00FF`, at least according to their docblocks, so I'm not sure how this ever worked in the first place. Too tired to sort it now, will come back to it tomorrow...